loader/loader.c

```diff
@@ -1078,7 +1074,7 @@ static size_t loader_platform_combine_path(char *dest, size_t len, ...) {
 
 // Given string of three part form "maj.min.pat" convert to a vulkan version number.
 static uint32_t loader_make_version(char *vers_str) {
```

Comment:
Is loader_make_version even needed anymore?

Reply:
It's used when parsing the JSON manifest files. Only gets used twice, but I'd say keeping it seems good.